RoadMap_SwiftUI/RoadMap_SwiftUI/ViewModel/AssortmentViewModel.swift

```diff
@@ -0,0 +1,66 @@
+//
+//  AssortmentViewModel.swift
+//  RoadMap_SwiftUI
+//
+//  Created by Артур Щукин on 26.01.2023.
+//
```

Comment:
```suggestion
//
//  AssortmentViewModel.swift
//  RoadMap_SwiftUI
//
//  Created by Артур Щукин on 26.01.2023.
//
```
не отработал форматер